src/apple/deps/xcode_plugin.rs

```diff
@@ -60,6 +60,7 @@ pub fn xcode_user_dir() -> Result<PathBuf, Error> {
 
 pub fn xcode_developer_dir() -> Result<PathBuf, Error> {
     duct::cmd("xcode-select", ["-p"])
+        .stdout_capture()
         .run()
```

Comment:
```suggestion
        .read()
```

Reply:
In this and following cases, I feel like it's important to know this is stdout or stderr.

Reply:
in this case we only care about stdout and we convert it to String, so `read` should be used here instead.